Sources/ArgumentParser/Completions/CompletionsGenerator.swift

```diff
@@ -164,11 +172,39 @@ extension ParsableCommand {
   }
 }
 
+extension [ParsableCommand.Type] {
+  // Include default 'help' subcommand in nonempty subcommand list iff no help subcommand already exists.
+  mutating func addHelpSubcommandIffMissing() {
```

Comment:
I know what "iff" means, but past experience tells me we'll get multiple PRs "correcting" this spelling. I think the meaning of just plain "if" is fine here instead (in the comment and the method name).

Reply:
Will replace. Or should I just include a comment like the following?

```swift
// "Iff" is not a misspelling; it means "if & only if"
```

Reply:
lets stick with "if"

Reply:
Pushed fix.